apps/veil/src/pages/tournament/server/delegator-history.ts

```diff
@@ -15,17 +15,17 @@ export type DelegatorHistorySortDirection = (typeof DIRECTIONS)[number];
 
 export interface DelegatorHistoryRequest {
   epochs: string[];
-  address: string;
   limit?: number;
   page?: number;
   sortKey?: DelegatorHistorySortKey;
   sortDirection?: DelegatorHistorySortDirection;
 }
 
 export interface TournamentDelegatorHistoryResponse {
+  address: Address;
+  total_items: number;
+  total_rewards: number;
```

Comment:
We should not let the database dictate our domain model. We should just map the result of the database query onto the convention here.